vm-setup/roles/v1aX_integration_test/tasks/upgrade.yml

```diff
@@ -147,26 +156,48 @@
       state: directory
       recurse: yes
     with_items:
+    - dev-repository/cluster-api/{{ CAPIRELEASE }}
+    - dev-repository/bootstrap-kubeadm/{{ CAPIRELEASE }}
+    - dev-repository/control-plane-kubeadm/{{ CAPIRELEASE }}
+    - overrides/infrastructure-metal3/{{ CAPIRELEASE }}
```

Comment:
These are not needed. 
The `cmd/clusterctl/hack/create-local-repository.py` is create the folders with the hard coded value. Then you copy theset to `CAPI_REL_TO_VERSION` folders